samza-core/src/main/java/org/apache/samza/clustermanager/ContainerManager.java

```diff
@@ -97,7 +152,10 @@ void handleContainerLaunch(SamzaResourceRequest request, String preferredHost, S
    */
   void handleContainerStop(String processorId, String containerId, String preferredHost, int exitStatus,
       Duration preferredHostRetryDelay, ContainerAllocator containerAllocator) {
-    if (standbyContainerManager.isPresent()) {
+    if (hasActiveContainerPlacementAction(processorId)) {
```

Comment:
```hasActiveContainerPlacementAction()``` should be defined either on the containerId or on requestId, since a processor can have multiple container-placement-actions during its lifetime.